docs/contributing/contributing.md

```diff
@@ -3,34 +3,76 @@ id: contributing
+1. Make sure your Content Pack is properly [documented](../documentation/pack-docs) and read the [documentation best practices](../documentation/documentation_tips).
+1. Validate your content: the validation hook should run automatically every time you `git commit`. You can also run the validation manually by using [demisto-sdk validate](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/validate/README.md). 
+1. As you build newer versions of your Content Pack, document your changes in a relevant release notes file as detailed [here](../documentation/release-notes).
+
+At this point you should be ready to submit a Pull Request! Check out again our [Contributing Checklist](../contributing/checklist), and for more details on the review process, refer to our [PR Conventions](../contributing/conventions) document.
+
+**Note**: if you are a technology partner, make sure you have reviewed the use cases with your [Cortex XSOAR Alliances Team](mailto:soar.alliances@paloaltonetworks.com) and that you have a *Partner ID* to associate your Pull Request to.
+
+A good working example that summarizes all of the above is the [Hello World Content Pack](https://github.com/demisto/content/tree/master/Packs/HelloWorld) that you can use as a reference. Also check out the [Hello World Design Document](https://docs.google.com/document/d/1wETtBEKg37PHNU8tYeB56M1LE314ux086z3HFeF_cX0).
 
- This document describes the main flow that covers most of the cases and summarizes everything you must do before and after opening a Pull Request on GitHub to contribute your pack.
+This guide doesn't cover all the topics: please browse the left sidebar and use the search bar to find what you need, and reach out for help over [Slack](https://start.paloaltonetworks.com/join-our-slack-community) on the `#demisto-developers` channel when in doubt.
 
-Before proceeding, sure to check the content contribution [checklist](../contributing/checklist) to make sure you have created everything you need.
 
 ## Before opening a Pull Request
 
 In order to be able to submit a Pull Request to the Cortex XSOAR [GitHub Repository](https://github.com/demisto/content), you need to:
 
-- Have a [GitHub](https://github.com) account that you'll use to create the fork and open the Pull Request.
-- Create a fork of the repo in your own Github account. You can follow the instruction described [GitHub docs](https://guides.github.com/activities/forking/) and use the examples from the [Development environment tutorial](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-2-fork-the-github-repo).
-- Create a new branch that will contain all of your contributed content. You can reference the [Create a branch](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-6-create-a-branch) step in the development setup toturial. 
+- Make sure to check the content contribution [checklist](../contributing/checklist) to make sure you have created everything you need.
```

Comment:
This is mentioned too many times. I think it is better as part of the `Guidelines`

Reply:
This is now mentioned only twice in the document: once in the guidelines and once here. I believe it's important to keep it in both places because users might read the two sections at different times.

Reply:
This is true but obviously, this applies to several items in these lists. I am just concerned about listing down too many items making this article too tedious and hard to follow. Obviously, one item is not going to change that.